lib/renditions/utils.ts

```diff
@@ -0,0 +1,38 @@
+import * as path from 'path';
+import { Component, ComponentRendition, ComponentsDefinition, GetFileContentType } from '../models';
+import merge = require('lodash.merge');
+import { deepFreeze } from '../util/freeze';
+
+export async function loadRenditions(
```

Comment:
Not sure, but shouldn't this be `loadHtmlRenditions`? as the PSV one is left out of scope.

Reply:
Makes sense.